crates/aptos/src/move_tool/mod.rs

```diff
@@ -109,7 +109,7 @@ impl MoveTool {
 
 const VAR_BYTECODE_VERSION: &str = "MOVE_BYTECODE_VERSION";
 
-fn set_bytecode_version(version: Option<u32>) {
+pub(crate) fn set_bytecode_version(version: Option<u32>) {
```

Comment:
Shouldn't we update this to use the right version for current network? We can still allow CLI users to override but if they don't, it should fallback to network-specific version.